sdk/communication/communication-messages-rest/samples-dev/MessageTemplate_List.ts

```diff
@@ -9,7 +9,7 @@ dotenv.config();
 async function main() {
     const credential = new AzureKeyCredential(process.env["ACS_ACCESS_KEY"] || "");
     const endpoint = process.env["ACS_URL"] || "";
-    const client:AzureCommunicationMessagesServiceClient = MessageTemplateClient(endpoint, credential);
+    const client:MessagesServiceClient = MessageTemplateClient(endpoint, credential);
```

Comment:
```suggestion
    const client = MessageTemplateClient(endpoint, credential);
```

We don't really need to have the type explicitly

Reply:
I fixed this.